### DISCUSSION TITLE
[Feature] alternativ video encoding

### DISCUSSION THREAD

**Discussion by erviker** [immich-app/immich#1862] [Feature Request]
### The feature

an option to do my own video encoding service
i'm already running Tdarr server and nodes as an video encoder for my media library.
who'd like to run this as the transcoder for video and not the docker host of immich.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by zackpollard** [comment]
We probably won't be looking to integrate with something like Tdarr, however we do have the idea of scaling out the system to other machines in our minds. We will likely make this possible in future updates by removing the reliance for the other containers to be connected to the same filesystem. You can already run the microservices/machine-learning on a separate server if you give it access to the same filesystem and setup the networking accordingly.
